src/profile-logic/marker-data.js

```diff
@@ -40,6 +40,7 @@ export function getJankMarkers(
 ): Marker[] {
   const addMarker = () =>
     jankInstances.push({
+      markerIndex: jankInstances.length,
```

Comment:
This is not used for jank markers but I made this property mandatory so that I don't forget to add it in all places where it's needed.